Vagrantfile

```diff
@@ -25,6 +25,7 @@ Vagrant.configure("2") do |config|
 
     cd /vagrant
     echo "localhost" >> hosts
+    ansible-galaxy install -r contrib/test/ci/requirements.yml
```

Comment:
hmm... I think we don't have `contrib/test/ci/requirements.yml`

Reply:
added the file